src/microsite/getting-started/index.html

```diff
@@ -98,7 +98,7 @@ <h2>Dynamic time-based modeling</h2>
 <p><img class="" style="float: right;" src="https://building-models-resources.concord.org/dynamic.png" alt="dynamic time-based model" width="314" height="200"><br>
 In dynamic time-based modeling, the output shows how the system changes over time. Notice how the graphs are now line graphs rather than bar graphs to indicate change over time rather than just a single value as represented by the bar. New types of variables and relationships become possible. Collectors “remember” their previous value and increase or decrease each step of the model calculation, while transfer variables can move amounts between collectors. Feedback becomes a prominent feature of many dynamic time-based models.</p>
 <h2>Tutorial</h2>
-<p>Try a step-by-step tutorial by opening this <a href="https://sagemodeler.concord.org/#file=examples:Getting%20Started" target="_blank" rel="noopener">Getting Started document</a> and following the instructions.</p>	</div>
+<p>Try a step-by-step tutorial by opening this <a href="https://sagemodeler.concord.org/#file=examples:Getting%20Started" target="_blank" rel="noopener">Getting Started document</a> and following the instructions. Also available <a href="https://sagemodeler.concord.org/app/?lang=es#file=examples:Gu%C3%ADa%20Comenzando%20-%20Getting%20started%20in%20Spanish" target="_blank" rel="noopener">en espa&ntilde;ol</a>.</p>	</div>
```

Comment:
I'm getting an "Unable to load" error when I try the link you've added (https://sagemodeler.concord.org/app/?lang=es#file=examples:Gu%C3%ADa%20Comenzando%20-%20Getting%20started%20in%20Spanish)

Reply:
@dougmartin Weird. I just checked it and it's working for me. Is it still not working for you? 

![image](https://user-images.githubusercontent.com/367459/96598494-ecbea200-12bc-11eb-9a70-f33fdea52070.png)

Reply:
Fixed with cache clear.  Looks good :+1: